src/ocean/THCM.C

```diff
@@ -219,44 +221,44 @@ THCM::THCM(Teuchos::ParameterList& params, Teuchos::RCP<Epetra_Comm> comm) :
                 << xdist, __FILE__, __LINE__);
     }
 
-    hdim         = paramList.get("Depth hdim", 4000.0);
-    double qz    = paramList.get("Grid Stretching qz", 1.0);
-    int  itopo   = paramList.get("Topography", 1);
-    bool flat    = paramList.get("Flat Bottom", false);
-    comp_sal_int = paramList.get("Compute salinity integral", true);
+    hdim         = paramList.get<double>("Depth hdim");
+    double qz    = paramList.get<double>("Grid Stretching qz");
+    int  itopo   = paramList.get<int>("Topography");
+    bool flat    = paramList.get<bool>("Flat Bottom");
+    comp_sal_int = paramList.get<bool>("Compute salinity integral");
 
-    bool rd_mask = paramList.get("Read Land Mask", false); //== false in experiment0
+    bool rd_mask = paramList.get<bool>("Read Land Mask"); //== false in experiment0
+    std::string mask_file = paramList.get<std::string>("Land Mask");
```

Comment:
This should be inside of the if statement. If `rd_mask` is false it should be unused.

Reply:
I had that initially, but then the test that nothing is unused fails (which I had to check that you guys get the right result).

Although I just realised a simple lifehack, I suppose I can just change the test parameter list to set "Read Land Mask" to true, then these *would* be use in the test...

Reply:
Yes, and if you really want to make sure `[unused]` also works you could set it to `false` and check that it's unused (but don't do the other check).